Jenkinsfile_nightly

```diff
@@ -85,11 +86,18 @@ withNightlyPipeline(type, product, component) {
   before('fullFunctionalTest') {
     env.PARALLEL_CHUNKS = params.FUNCTIONAL_TESTS_WORKERS
     env.MOCKED_PAYMENTS = false
+    env.TESTS_FOR_ACCESSIBILITY = true
```

Comment:
tbh, i would prefer to follow existing naming patterns, we have e.g

    env.SKIP_API_TESTS
    env.SKIP_BROWSER_TESTS